packages/runtime/datastore/src/dataStoreRuntime.ts

```diff
@@ -429,17 +429,17 @@ IFluidDataStoreChannel, IFluidDataStoreRuntime, IFluidHandleContext {
         });
 
         this.localChannelContextQueue.clear();
-        this.bindToContext();
+        this.updateBindState();
```

Comment:
Can we instead inline the call and get rid of this.bindState? I think (given that's the only entry point) that we ensure it's called only once, so no need to track it? In places where it's tested, can we leverage appropriate this.graphAttachState states?

Reply:
Good point. Removed it. Can you please take a look if the logic looks correct now.

Reply:
Looks good to me - one less concept to think about!

Reply:
Yes, exactly. Now we just need to rename graphAttachState!